docs/blog/newsletter-may-2017.rst

```diff
@@ -0,0 +1,111 @@
+
+Some approaches were pretty lightweight and focused on adding clarity without introducing any coding behind the scenes:
+
+* Choose a formatting convention for any example data, so the replaceable parts of the snippet are easy to pick out visually.
+* Build up a small cast of fictional characters that you can write about throughout your docs. This lets you use a realistic name and password combo that your readers will have some context for.
+* Expand on the basic 'username' and 'password' with more expressive strings like, 'example_username', 'your_username', 'replace_with_username', and so on
+
+If you're looking for more dynamic solutions to the same problem, we've got some good options for you on that front as well:
+
+* Automate the formatting for replaceable content by defining your own classes. DocBook, for example, will let you do this with the <replaceable> tag, and for Markdown, the same effect can be achieved with some help from `hercule <https://github.com/jamesramsay/hercule>`_ and bash scripts.
+* If you're using fictional characters, you can take it a step further and create actual accounts for them in your system. Then, whenever your real-life users try to execute something with a fictional account's data, you can throw an error message to prompt them to swap in their actual details.
+* For maximum coolness, you could build in a prompt at the top of your docs asking for the relevant details, then use that info to generate personalized code samples that are ready to go, right out of the gate. With this option, there are some authentication and security concerns to address, but `Stripe's API docs <https://stripe.com/docs>`_ are a great example of how this might look.
+
+------------------------------------------
+
+The Challenges of Documentation in an Agile Environment
+#######################################################
+
+How can writers work with Agile dev teams? It's a perennial question that arose again this month and produced some excellent discussion. TL;DR: there are many models, and writers have to figure out what works best for their particular teams, processes, and products. Here's a round-up of the approaches that the WtD community came up with:
+
+* With teams that perform regression testing and user acceptance testing (UAT) after a scheduled sprint, docs are sometimes developed during these post-sprint cycles. In this kind of workflow, docs and testing are often closely allied.
+
+* Some writers are embedded on Agile dev teams, attend standups, and work on the regular sprint cycle, with stories assigned for doc; others work with parallel sprints, and devs/PMs serve as stakeholders.
+
+* Some of the "embedded" writers document features at the start of development, but doc is not released until feature complete. Other writers find that features change too much during dev cycles for this approach to be worthwhile.
```

Comment:
let's flesh out the second half of this first sentence: '...but the doc is not released until the feature is complete.'